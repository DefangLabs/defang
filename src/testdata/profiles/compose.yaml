services:
  always:
    image: alpine

  never:
    image: alpine
    profiles:
      - never

  defangonly:
    image: alpine
    profiles:
      - defang
