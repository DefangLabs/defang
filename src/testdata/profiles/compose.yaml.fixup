{
  "always": {
    "command": null,
    "entrypoint": null,
    "image": "alpine",
    "networks": {
      "default": null
    }
  },
  "defangonly": {
    "profiles": [
      "defang"
    ],
    "command": null,
    "entrypoint": null,
    "image": "alpine",
    "networks": {
      "default": null
    }
  }
}