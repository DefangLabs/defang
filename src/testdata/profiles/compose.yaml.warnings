 ! service "always": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors
 ! service "defangonly": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors