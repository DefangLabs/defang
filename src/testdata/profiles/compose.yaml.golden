name: profiles
services:
  always:
    image: alpine
    networks:
      default: null
  defangonly:
    profiles:
      - defang
    image: alpine
    networks:
      default: null
networks:
  default:
    name: profiles_default
